Source/DataTypes/SvgUnitCollection.cs

```diff
@@ -77,45 +80,43 @@ public override object ConvertFrom(ITypeDescriptorContext context, CultureInfo c
         {
             if (value is string)
             {
-                var s = ((string)value).Trim();
-
-                if (s.Equals("inherit", StringComparison.OrdinalIgnoreCase))
-                    return null;
-
+                var points = ((string)value).Trim().Split(new char[] { ',', ' ', '\r', '\n', '\t' }, StringSplitOptions.RemoveEmptyEntries);
                 var units = new SvgUnitCollection();
-                units.IsNull = s.Equals("none", StringComparison.OrdinalIgnoreCase);
-                if (!units.IsNull)
-                    foreach (var point in s.Split(new char[] { ',', ' ', '\r', '\n', '\t' }, StringSplitOptions.RemoveEmptyEntries))
-                    {
-                        var newUnit = (SvgUnit)_unitConverter.ConvertFrom(point.Trim());
-                        if (!newUnit.IsNone)
-                            units.Add(newUnit);
-                    }
+
+                foreach (var point in points)
+                {
+                    var newUnit = (SvgUnit)_unitConverter.ConvertFrom(point.Trim());
+                    if (!newUnit.IsNone)
+                        units.Add(newUnit);
+                }
 
                 return units;
             }
 
             return base.ConvertFrom(context, culture, value);
         }
+    }
 
-        public override bool CanConvertTo(ITypeDescriptorContext context, Type destinationType)
-        {
-            if (destinationType == typeof(string))
-                return true;
-
-            return base.CanConvertTo(context, destinationType);
-        }
-
-        public override object ConvertTo(ITypeDescriptorContext context, CultureInfo culture, object value, Type destinationType)
+    internal class SvgStrokeDashArrayConverter : SvgUnitCollectionConverter
+    {
+        public override object ConvertFrom(ITypeDescriptorContext context, CultureInfo culture, object value)
         {
-            if (value != null && destinationType == typeof(string))
+            if (value is string)
             {
-                if (((SvgUnitCollection)value).IsNull)
-                    return "none";
-                return ((SvgUnitCollection)value).ToString();
+                var s = ((string)value).Trim();

```

Comment:
Only [stroke-dasharray](https://www.w3.org/TR/SVG11/painting.html#StrokeProperties) can set none and inherit.